### ISSUE TITLE
Message:[AppGalleryConnectPublishService]file url is invalidate

### ISSUE THREAD

**Issue by pamegonzalez** [FutureMind/bitrise-step-huawei-app-gallery-apk-distribution#4]
Hi, I'm facing this issue

INFO[04:43:46]  * [OK] Step dependency (jq) installed, available. 
Apk path is: 
Lang is: es-419
Obtaining a Token
Obtaining a Token - Done
Obtaining the File Upload URL
Obtaining the File Upload URL - Done
Uploading a File
Uploading a File - Done
Updating App File Information - add previoulsy uploaded file
FileName: build_235.apk
Updating App File Information - Done
**Message:[AppGalleryConnectPublishService]file url is invalidate
Code: 204144641**

What am I doing wrong? I tried to look for this issue, but I couldn't find anything.
Thanks

**Comment by klamborowski** [comment]
Only information about this code I've found there : https://developer.huawei.com/consumer/en/doc/development/AppGallery-connect-References/agcapi-returncode_v2

Few moments ago I've made a new PR #5 with debug logs, it could be a bit help for your debugging.

What's more try to post an issue to [Huawei Support ](https://developer.huawei.com/consumer/en/support/feedback/#/) in AppGallery Connect -> Connect API category.

**Comment by klamborowski** [comment]
@pamegonzalez Version 1.0.1 with debug logs is live now. Try to build with debug and maybe you can get some more information on that issue :)

![obraz](https://user-images.githubusercontent.com/4972310/87572984-9bef4a00-c6cc-11ea-852c-eef309861cfe.png)

**Comment by pamegonzalez** [comment]
> @pamegonzalez Version 1.0.1 with debug logs is live now. Try to build with debug and maybe you can get some more information on that issue :)
> 
> ![obraz](https://user-images.githubusercontent.com/4972310/87572984-9bef4a00-c6cc-11ea-852c-eef309861cfe.png)

Nice! Thanks! I'll try this, this step is failing for me still.

**Comment by ferPrieto** [comment]
@pamegonzalez did you sort out the issue you mentioned?

**Comment by pamegonzalez** [comment]
> @pamegonzalez did you sort out the issue you mentioned? 

No, actually. I'm making our deployments to App Gallery by hand :(.
I haven't test this again, I'll give it another try.
Are you having a similar trouble?

**Comment by ferPrieto** [comment]
@pamegonzalez  Please have a look at the official[ Deploy to AppGallery Step](https://www.bitrise.io/integrations/steps/appgallery-deploy).
I've released the latest version today (1.5.0), if you find any issue you can post it [here](https://github.com/ferPrieto/steps-app-gallery-deploy)

**Comment by pamegonzalez** [comment]
> @pamegonzalez Please have a look at the official[ Deploy to AppGallery Step](https://www.bitrise.io/integrations/steps/appgallery-deploy).
> I've released the latest version today (1.5.0), if you find any issue you can post it [here](https://github.com/ferPrieto/steps-app-gallery-deploy)

Now it works like a charm ❤️ 
Thanks for improving it!!
